lib/build-view.js

```diff
@@ -17,6 +17,7 @@ module.exports = (function() {
     this.titleLoopIndex = 0;
     this.a2h = new Convert();
     this.monocle = false;
+    this.attach();
```

Comment:
Is you acutely pointed out, when we call `atom.config.observe` with the keepVisible argument, `visibleFromConfig` will be called with the current choice the user have. This attach is not necessary then, because it will be either attach or detach att that point.